Makefile

```diff
@@ -16,8 +16,11 @@ CPPFLAGS += -DNDEBUG -D_REENTRANT					\
 	-Ispidermonkey/src -Ispidermonkey/src/build			\
 	-I/usr/include							\
 
-
-PY_PYTHON=$(shell python -c "import sys; print(sys.executable)")
+PY_EXEC=$(shell which python2)
+ifndef PY_EXEC
+	PY_EXEC=python
```

Comment:
Would it be worth trying to use `python2.7` first, and then if that fails, trying `python2` and then only if that fails using `python`? Fwiw, on macOS 10.13.6 at least, there's a `python2.7` but no `python2`. I'm not sure if that really matters, but I thought I'd point it out.

Reply:
Ah good point. I'll try that.